buildSrc/src/main/groovy/wala-java.gradle

```diff
@@ -27,9 +27,18 @@ repositories {
 // only use Error Prone on JDK 9+ JVMs
 final useErrorProne = JavaVersion.current().isJava9Compatible()
 
+// checks if NullAway should be used.
+def useNullAway(project) {
```

Comment:
Use `final` here instead of `def`: we don't expect to reassign `useNullAway` to have a different value other than the function being created here.